### ISSUE TITLE
Crash at MobileNet's second forward at batch_norm1's activation when only relu and softmax were ported

### ISSUE THREAD

**Issue by zhaofenqiang** [zhaofenqiang/PaddleOnACL#2]
Only port the ActivationLayer, including ReLU and softmax funciton. 
First forward was normal, second forward crashed.

Program ouput:

> zfq@zfq-pi:~/mobile$ ./inference --merged_model ./mobilenet_flowers102.paddle --input_size 150528
I0411 06:22:18.676442 32307 Util.cpp:166] commandline:  
Time of init paddle 2.93423 ms.
Time of create from merged model file 710.877 ms.
image
__conv_0__
__batch_norm_0__
running relu forward...
running on acl...
__conv_1__
__batch_norm_1__
running relu forward...
running on acl...
__conv_2__
__batch_norm_2__
running relu forward...
running on acl...
__conv_3__
__batch_norm_3__
running relu forward...
running on acl...
__conv_4__
__batch_norm_4__
running relu forward...
running on acl...
__conv_5__
__batch_norm_5__
running relu forward...
running on acl...
__conv_6__
__batch_norm_6__
running relu forward...
running on acl...
__conv_7__
__batch_norm_7__
running relu forward...
running on acl...
__conv_8__
__batch_norm_8__
running relu forward...
running on acl...
__conv_9__
__batch_norm_9__
running relu forward...
running on acl...
__conv_10__
__batch_norm_10__
running relu forward...
running on acl...
__conv_11__
__batch_norm_11__
running relu forward...
running on acl...
__conv_12__
__batch_norm_12__
running relu forward...
running on acl...
__conv_13__
__batch_norm_13__
running relu forward...
running on acl...
__conv_14__
__batch_norm_14__
running relu forward...
running on acl...
__conv_15__
__batch_norm_15__
running relu forward...
running on acl...
__conv_16__
__batch_norm_16__
running relu forward...
running on acl...
__conv_17__
__batch_norm_17__
running relu forward...
running on acl...
__conv_18__
__batch_norm_18__
running relu forward...
running on acl...
__conv_19__
__batch_norm_19__
running relu forward...
running on acl...
__conv_20__
__batch_norm_20__
running relu forward...
running on acl...
__conv_21__
__batch_norm_21__
running relu forward...
running on acl...
__conv_22__
__batch_norm_22__
running relu forward...
running on acl...
__conv_23__
__batch_norm_23__
running relu forward...
running on acl...
__conv_24__
__batch_norm_24__
running relu forward...
running on acl...
__conv_25__
__batch_norm_25__
running relu forward...
running on acl...
__conv_26__
__batch_norm_26__
running relu forward...
running on acl...
__pool_0__
__fc_layer_0__
running softmax act....
running on acl...
Time of first run time 2969.11 ms.
image
__conv_0__
__batch_norm_0__
running relu forward...
running on acl...
Thread [1995748352] Forwarding __batch_norm_0__, 
*** Aborted at 1523442142 (unix time) try "date -d @1523442142" if you are using GNU date ***
PC: @        0x0 (unknown)
*** SIGSEGV (@0x1) received by PID 32307 (TID 0x76f4b400) from PID 1; stack trace: ***
    @ 0x767d7270 (unknown)
    @ 0x75cf4ece arm_compute::Iterator::Iterator()
    @ 0x75cf9170 arm_compute::NEActivationLayerKernel::activation<>()
    @ 0x75a2b350 arm_compute::CPPScheduler::schedule()
    @ 0x76bb0a5a paddle::ACLOperator::run()
    @ 0x76bb0c6a paddle::ACLOperator::acl_run()
    @ 0x76bb1948 paddle::reluActivation::forward()
    @ 0x76aade38 paddle::Layer::forwardActivation()
    @ 0x76b21c5e paddle::BatchNormalizationLayer::forward()
    @ 0x76b4c91e paddle::NeuralNetwork::forward()
    @ 0x76a17e06 paddle_gradient_machine_forward
    @    0x1168c infer()
    @    0x11d28 main
    @ 0x767c88aa __libc_start_main
Segmentation fault
